cli/src/main.rs

```diff
@@ -64,14 +63,30 @@ fn run() -> error::Result<()> {
                 .arg(Arg::with_name("debug").long("debug").short("d"))
                 .arg(Arg::with_name("debug-graph").long("debug-graph").short("D")),
         )
+        .subcommand(
+            SubCommand::with_name("highlight")
+                .about("Highlight a file")
+                .arg(
+                    Arg::with_name("path")
+                        .index(1)
+                        .multiple(true)
+                        .required(true),
+                )
+                .arg(Arg::with_name("html").long("html").short("h")),
+        )
         .get_matches();
 
     let home_dir = dirs::home_dir().unwrap();
     let current_dir = env::current_dir().unwrap();
     let config_dir = home_dir.join(".tree-sitter");
+    let theme_path = config_dir.join("theme.json");
+    let parsers_dir = config_dir.join("parsers");
 
-    fs::create_dir_all(&config_dir).unwrap();
-    let mut loader = Loader::new(config_dir);
+    // TODO - make configurable
+    let parser_repo_paths = vec![home_dir.join("github")];
```

Comment:
Here I've hard coded the CLI to look for parsers in the `~/github` folder. I could probably allow this list of paths to be specified in a config file in the `~/.tree-sitter` folder instead. I just haven't decided how I want that config file to look yet.